TF_Service_dotNet/TouchFree_Service/Program.cs

```diff
@@ -24,7 +31,7 @@ static void Main(string[] args)
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
                     webBuilder.UseStartup<Startup>();
-                    webBuilder.UseUrls("http://localhost:9739");
+                    webBuilder.UseUrls("http://" + config.ServiceIP + ":" + config.ServicePort);
```

Comment:
Can you change this one to use string interpolation:
`$"http://{config.ServiceIP}:{config.ServicePort}"`

Reply:
Sure - would you rather this on the Unity side too?